elementary/monitor/dbt_project/macros/base_queries/current_tests_run_results_query.sql

```diff
@@ -1,21 +1,23 @@
-{% macro current_tests_run_results_query(days_back = none) %}
+{% macro current_tests_run_results_query(days_back = none, invocation_id = none) %}
     {% set columns = adapter.get_columns_in_relation(ref('elementary', 'elementary_test_results')) %}
     {% set columns_names = [] %}
     {% for column in columns %}
         {% do columns_names.append(column.column | lower) %}
     {% endfor %}
     {% if 'test_short_name' not in columns_names or 'test_alias' not in columns_names %}
-        {{ elementary_internal.current_tests_run_results_backcomp(days_back=days_back) }}
+        {{ elementary_internal.current_tests_run_results_backcomp(days_back=days_back, invocation_id=invocation_id) }}
     {% else %}
-        {{ elementary_internal.current_tests_run_results(days_back=days_back) }}
+        {{ elementary_internal.current_tests_run_results(days_back=days_back, invocation_id=invocation_id) }}
     {% endif %}
 {% endmacro %}
 
 
-{% macro current_tests_run_results(days_back = none) %}
+{% macro current_tests_run_results(days_back = none, invocation_id = none) %}
     with elementary_test_results as (
         select * from {{ ref('elementary', 'elementary_test_results') }}
-        {% if days_back %}
+        {% if invocation_id %}
+            where invocation_id = {{ "'" ~ invocation_id ~ "'"}}
```

Comment:
Not critical, but Elon convinced me that using python string formatting is clearer than the Jinja concat syntax :)
(e.g. `'{}'.format(invocation_id)`)

Reply:
Agree!